readthedocs/basic/signing-in.rst

```diff
@@ -94,16 +94,21 @@ Signing In as a Bot Account
 ===========================
 
 You can also use Telethon for your bots (normal bot accounts, not users).
-You will still need an API ID and hash, but the process is very similar:
+You will still need an API ID and hash that could be retrived from *any application*
```

Comment:
I find this "from any application" wording confusing. What application?

I think it would be better to explicitly mention that the API ID comes from my.telegram.org, and the bot token is a completely separate thing coming from `@BotFather`, but both are needed. (With appropriate links to both sites.)

People also seem to be confused if every user of their application needs to provide their own API ID (no, they don't, the application uses the developer's API ID, which for someone reading the documentation, they are the developer and they have to obtain and provide their own API ID), but I don't know if it's a good idea to also explain that here (it might get too long or confusing).